src/main/java/dao/JanggiConnection.java

```diff
@@ -0,0 +1,22 @@
+package dao;
+
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class JanggiConnection {
+    private static final String SERVER = "localhost:13306";
+    private static final String DATABASE = "janggi";
+    private static final String OPTION = "?useSSL=false&allowPublicKeyRetrieval=true&serverTimezone=UTC";
+    private static final String USERNAME = "root";
+    private static final String PASSWORD = "root";
+
+    public java.sql.Connection getConnection() {
```

Comment:
Connection 을 import하지 않는 이유가 있으신가요?
```suggestion
    public Connection getConnection() {
```

Reply:
앗 해당 객체의 이름이 처음에 Connection 이었어서 이렇게 됐었는데 수정을 못했었네요 😓